spec/order_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'order'
+
+
+
+describe Order do
+
+  menu = {Hamburger: 1}
+
+  describe '#order_input'do
+
+    it 'allows you to input your order'do
+      expect(subject.order_input("Hamburger", 1, menu)).to eq([[:Hamburger],[1]])
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)